### DISCUSSION TITLE
Specific GIF will not render in an issue comment

### DISCUSSION THREAD

**Discussion by samatcodeapprove** [community/community#66083] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

See:
https://github.com/codeapprove/meta/issues/35

If I post the following markdown, the image is not rendered at all:
```
Lol, is this you?![](https://i.imgur.com/plJGnCb.gif)
```

The same markdown works with a different gif link:
```
Lol, is this you?![](https://media.tenor.com/HycmZ_R5ka8AAAAC/some-mascots-bounce-fail.gif)
```

For some reason GitHub can't render the Hackerman GIF ... I can't figure out why.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by arafajs** [comment]
A few things to try:

- Double check that the gif loads correctly when you visit the direct imgur link in a browser. If it doesn't load on its own, that could be why it's not rendering in GitHub.
- Try re-uploading the gif to a different host like Giphy or Tenor and using that link instead. Sometimes certain files don't play nicely with GitHub's markdown renderer.
- If the gif loads fine outside of GitHub, it could be an issue with the size or type of gif. Try optimizing it with a tool like GIFOptimizer or uploading a version that's under the GitHub file size limit.
- As a workaround, you could link to the gif image instead of trying to embed it. For example:

```
Lol, is this you? [See gif here](https://i.imgur.com/plJGnCb.gif)
```

**Comment by samatcodeapprove** [comment]
@arafajs thanks for the tips but I think this is purely a bug in GitHub. If you visit https://i.imgur.com/plJGnCb.gif you'll see that the gif loads fine. I did not create this gif so re-uploading it to a new host is not really an option.

GitHub should either render the image or render some sort of error that it can't. Rendering nothing is confusing.
